spec/models/item_spec.rb

```diff
@@ -130,5 +130,29 @@
       order.item_orders.create(item: @chain, price: @chain.price, quantity: 2)
       expect(@chain.no_orders?).to eq(false)
     end
+
+    it 'item#fulfillable?(num)' do
```

Comment:
Love what you are doing here! This really makes it easy to know what's being tested.

Reply:
Agreed. I'd be good with applying this naming convention across all model tests.